gap/attr.gi

```diff
@@ -530,14 +530,7 @@ function(D)
 end);
 
 InstallMethod(DigraphNrEdges, "for a digraph", [IsDigraphByOutNeighboursRep],
-function(D)
-  local m;
-  m := DIGRAPH_NREDGES(D);
-  if IsImmutableDigraph(D) then
-    SetIsEmptyDigraph(D, m = 0);
```

Comment:
TODO: nonempty digraphs now automatically know that they are nonempty once they know they number of edges, because of a true method, but empty digraphs do not know that they are empty, even if their number of edges is zero. So perhaps I want to add something like this back in (or find another way of doing it).

Reply:
Probably I just want there to be an `IsEmptyDigraph` immediate method for `HasDigraphNrEdges`, doing the same (well, opposite) as the new immediate method for `DigraphHasEdges`.